chainercv/transforms/image/random_expand.py

```diff
@@ -24,6 +24,9 @@ def random_expand(img, max_ratio=4, fill=0, return_param=False):
             paper, this value is 4.
         fill (float, tuple or ~numpy.ndarray): The value of padded pixels.
```

Comment:
Let me summarize.

`fill` can be one of `float, tuple or (C, 1, 1)`.
`mean` is `(C, 1, 1)`.